mvc_controllers/comments.controller.js

```diff
@@ -0,0 +1,15 @@
+const {insertComment} = require('../mvc_models/comments.model')
+const {checkArticleExists} = require('../mvc_models/articles.model')
+exports.postComment = (req, res, next) => {
+    const newComment = req.body
+    const id = req.params.article_id
+    checkArticleExists(id)
```

Comment:
why do you need to check if it exists first? What happens if you post to a non-existent article. You'll likely find Postgres won't be happy.